app/lib/typesAndInterfaces.ts

```diff
@@ -105,6 +105,10 @@ export interface StaveNoteData {
   staveNoteAbsoluteX: number;
   userClickY: number;
 }
+export interface Clef {
```

Comment:
Clef is already the name of a VexFlow class, so you need to come up with a more original name for types like these.